server/src/main/resources/application.conf

```diff
@@ -12,7 +12,7 @@ application {
   websocket.keepAliveInterval = 30s
 
   translations {
-    cacheKey = translations
+    cacheKey = translations_large
```

Comment:
Currently the image hashes aren't overridden. Changing this to write to a new cache key to have a fresh start.